kube/helm/helm.go

```diff
@@ -113,6 +105,11 @@ func (hc *Actions) Get(name string) (*release.Release, error) {
 
 // ListReleases lists chart releases
 func (hc *Actions) ListReleases() ([]compose.Stack, error) {
+	logger := func(format string, v ...interface{}) {}
```

Comment:
We should remove this and use nil or set the default noop logger when building hc